app/controllers/mixins/ems_common_angular.rb

```diff
@@ -478,6 +478,9 @@ def get_hostname_from_routes(ems, endpoint_hash, token)
     rescue KubeException, OpenSSL::SSL::SSLError => e
       $log.warn("MIQ(#{controller_name}_controller-#{action_name}): get_hostname_from_routes error: #{e}")
       nil
+    rescue StandardError
```

Comment:
you can write just `rescue`, it default to StandardError.
and you'll need `=> e` to log the error.